packages/list-store/src/useListStore.test.ts

```diff
@@ -0,0 +1,142 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import {renderHook, act} from '@testing-library/react';
+import {useListStore} from './useListStore';
+import {SortBy} from './sort';
+
+describe('pageChange', () => {
+  it('will add the `limit` to the `offset` if the new `offset` is not greater than the `totalCount`', () => {
```

Comment:
For consistency with the other tests, use an active voice? For example: `it('adds the limit to...')`